docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md

```diff
@@ -299,7 +299,7 @@ TensorFlow*-specific parameters:
                         TensorFlow*: comma separated list of shared libraries
                         with TensorFlow* custom operations implementation.
   --disable_nhwc_to_nchw
-                        Disables default translation from NHWC to NCHW
+                        Disables default translation from NHWC to NCHW for old API
```

Comment:
@andrew-zaytsev How we can better explain that this option takes effect only for "old" API?

Reply:
@mvafin [DEPRECATED] Disables default translation from NHWC to NCHW. Since 2022.1 this option is deprecated and used only to maintain backward compatibility with previous releases.